_layouts/side-nav.html

```diff
@@ -1,55 +1,18 @@
 {% include head.html %}
 {% include top-navigation-bars.html %}
 
-
-<div class="page-header">
-	<h2 class="page-title">{{ page.title }}</h2>
-</div>
-
-
 <div class="page-content">
 
-	<nav class="side-menu">
+	{% include side-nav.html %}
 
-		<!-- Assign posts based on which page we're on and set the navigation title -->
-		{% if page.url contains "glossary" %}
-			{% assign nav-items = site.glossary | sort:"nav-position" %}
-		{% elsif page.url contains "tutorials" %}
-			{% assign nav-items = site.tutorials | sort:"nav-position" %}
-		{% elsif page.url contains "getting-started" %}
-			{% assign nav-items = site.getting-started | sort:"nav-position" %}
-			<h1>Getting Started</h1>
+	<main>
+		<h1 class="">{{ page.title }}</h1>
+		{% if page.tag != nil %}
+			<code class="wdk-tag">{{ page.tag }}</code>
```

Comment:
Is there a reason `{{ page.tag| xml_escape }}` wouldn't work here instead of the js solution?
src: http://jekyllrb.com/docs/templates/